addons/html_editor/static/src/editor/list/list_plugin.js

```diff
@@ -268,6 +261,7 @@ export class ListPlugin extends Plugin {
             this.indentLI(li);
         }
         restoreCursor();
+        // @todo pass a root argument to sanitize
```

Comment:
this should call a history step instead as it will sanitize it.
`// @todo @phoenix use history step instead`

Reply:
Changed comment as requested.